src/mainwindow/mainwindow.ui

```diff
@@ -432,30 +432,37 @@
         </property>
        </widget>
       </item>
-      <item row="4" column="1">
-       <widget class="QPushButton" name="pogiBrowseButton">
+      <item row="3" column="0" colspan="2">
+       <widget class="QLabel" name="label_19">
         <property name="text">
-         <string>Browse</string>
+         <string>CV POGI FILE:</string>
+        </property>
+        <property name="alignment">
+         <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="pogiFileNameEdit"/>
       </item>
+      <item row="4" column="1">
+       <widget class="QPushButton" name="pogiBrowseButton">
+        <property name="text">
+         <string>Browse</string>
+        </property>
+       </widget>
+      </item>
       <item row="5" column="0" colspan="2">
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Set POGI JSON File</string>
         </property>
        </widget>
       </item>
-      <item row="3" column="0" colspan="2">
-       <widget class="QLabel" name="label_19">
+      <item row="6" column="0">
+       <widget class="QPushButton" name="updateWidget">
```

Comment:
Following with the above comment, update the name of this button to testUpdateWidget